README.md

```diff
@@ -1,2 +1,23 @@
 # web-crawler
-Program designed to be used in Wikipedia sites only. It searches for the first link in the page and stop when it gets to a specific page (ex. philosophy)
+  Program designed to be used in Wikipedia sites only. It searches for the first link in the page and stop when it gets to a specific page (eg. philosophy)
```

Comment:
A program designed to be used for Wikipedia sites only. 

It searches for the first article link on any entry page and navigates through until it reaches the desired target page. 

(eg. Random article > navigation algorithm > Desired article)